tapcfg/server.go

```diff
@@ -301,6 +294,16 @@ func genServerConfig(cfg *Config, cfgLogger btclog.Logger,
 		},
 	)
 
+	// TODO(jhb): expose `SyncerActive` setting elsewhere
+	addrBook := address.NewBook(address.BookConfig{
+		Store:        tapdbAddrBook,
+		Syncer:       universeFederation,
+		SyncerActive: true,
```

Comment:
Why's this needed? Can't we just say if the passed interface is nil or not?

Reply:
Ah sure agree re being able to opt out of it, in which case, I think the server would just not set `Syncer` at all. Then if that's nil, all the new behavior is skipped.